polar-core/src/error.rs

```diff
@@ -439,6 +447,9 @@ impl ValidationError {
             ResourceBlock { term, .. }
             | SingletonVariable { term, .. }
             | UndefinedRuleCall { term }
+            | DuplicateResourceBlockDeclaration {
+                declaration: term, ..
```

Comment:
Not an issue introduced in this PR, but for 'relation' declarations the lexical context is going to point to the relation type (the value `Bar` in the `relations = { foo: Bar };` declaration; not the key `foo`).